src/components/my-career/AddJobRole.jsx

```diff
@@ -13,9 +14,9 @@ import SearchJobRole from './SearchJobRole';
 import SkillsQuizImage from '../../assets/images/skills-quiz/skills-quiz.png';
 
 const addIcon = () => (
-  <Icon
+  <Plus
     id="add-job-role-icon"
-    className="fa fa-plus add-job-icon"
+    className="add-job-icon"
     screenReaderText="Add Role"
```

Comment:
[curious] Does the `Plus` icon used as a React component accept as `screenReaderText` component? Is this intended to use `Icon` component still, but pass `Plus` as its `src` prop?